shell/protoc.sh

```diff
@@ -152,6 +152,10 @@ if has_feature "validation"; then
   )
 fi
 
+delete_validate() {
+  sed -i '' '/validate_pb/d' "$1" 2>/dev/null || sed -i '/validate_pb/d' "$1"
```

Comment:
Missing argument to `-i` in second half:
```suggestion
  sed -i '' '/validate_pb/d' "$1" 2>/dev/null || sed -i '' '/validate_pb/d' "$1"
```

Reply:
The linux version actually doesn't take a second half. That's why this is trying to do this messy `||` logic so it supports both mac and linux.